_commands/basics/kill.md

```diff
@@ -3,22 +3,129 @@
 
 kill
 --
-TODO: Add documentation for this command by submitting a pull request.
+Kill is a command that ends a process by specifying its PID (process ID), either via a signal or forced termination. 
 <!-- one line explanation would go here -->
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+List the running process
+
+$ ps
+PID TTY TIME CMD
+1293 pts/5 00:00:00 MyProgram
+
+Then Kill it
+
+$ kill 1293
+[2]+ Terminated MyProgram
 ~~~
 
 <!--more-->
 
+### Note
+
+The kill command has a misleading name because it does not actually kill processes. Rather, it sends signals to them. Each process is supplied with a set of standard signal handlers by the operating system in order to deal with incoming signals. When no signal is explicitly included in the command, signal 15, named SIGTERM, is sent by default. If this fails, the stronger signal 9, called SIGKILL, should be used.
```

Comment:
make sure people know that the lower the signal number, the stronger the kill signal will be.